docs/coding-guidelines/adding-api-guidelines.md

```diff
@@ -24,20 +24,20 @@ the implementation without compat concerns in future releases.
 
 ### Determine target framework
 
-`netstandard` or `netcoreapp` is the target framework version currently under development.
+`netstandard2.1` or `netcoreapp5.0` is the target framework version currently under development.
```

Comment:
`netstandard2.1` is already released, it's not under development.

Reply:
I think we can remove most of the netstandard mentions here as going forward net5 (netcoreapp5.0) should replace all these considerations.

Reply:
Agree.

---
In reply to: [377837257](https://github.com/dotnet/runtime/pull/32062#discussion_r377837257) [](ancestors = 377837257)